Formula/mariadb-connector-odbc.rb

```diff
@@ -0,0 +1,60 @@
+class MariadbConnectorOdbc < Formula
+  desc "MariaDB Standardized database driver"
+  homepage "https://downloads.mariadb.org/connector-odbc/"
+  # MariaDB's download page doesn't let you download without going through the site, probably a cookie-based check
+  # The connector is packaged at http://archive.mariadb.org/connector-odbc-2.0.13/source/mariadb-connector-odbc-2.0.13-ga-src.tar.gz, but there's no https here
+  # So the next best choice seems to be github:
+  url "https://github.com/MariaDB/mariadb-connector-odbc/archive/2.0.13.tar.gz"
```

Comment:
It's fine to download from github if it's the official project, so you can remove the comment